hadoop-yarn-project/hadoop-yarn/pom.xml

```diff
@@ -246,4 +246,5 @@
     <module>hadoop-yarn-ui</module>
     <module>hadoop-yarn-csi</module>
   </modules>
+  <!--  -->
```

Comment:
nit: Next time, see if you can make a change that doesn't leave behind detritus!